docs/users-manual/submitting-a-job.rst

```diff
@@ -853,28 +851,27 @@ Rank Expression Examples
 
 :index:`examples<single: examples; rank attribute>`
 :index:`rank examples<single: rank examples; ClassAd attribute>`
-:index:`rank<single: rank; submit commands>`
 
 When considering the match between a job and a machine, rank is used to
 choose a match from among all machines that satisfy the job's
 requirements and are available to the user, after accounting for the
 user's priority and the machine's rank of the job. The rank expressions,
 simple or complex, define a numerical value that expresses preferences.
 
-The job's ``Rank`` expression evaluates to one of three values. It can
-be UNDEFINED, ERROR, or a floating point value. If ``Rank`` evaluates to
+The job's :subcom:`Rank` expression evaluates to one of three values. It can
+be UNDEFINED, ERROR, or a floating point value. If :subcom:`Rank` evaluates to
```

Comment:
```suggestion
The job's :subcom:`rank` expression evaluates to one of three values. It can
be UNDEFINED, ERROR, or a floating point value. If :subcom:`rank` evaluates to
```